arbitrator/prover/src/machine.rs

```diff
@@ -2567,10 +2629,14 @@ impl Machine {
             |v| v.serialize_for_proof(),
         ));
 
-        out!(prove_window(
```

Comment:
proving a multistack should be:
proving the actual used stack as normal (either master or last cothread)
hash of the other stack (master, last cothread or 32-byte zeroes)
other_cothreads hash as defined in hashin above

Reply:
changed, ptal